mlir/test/Conversion/MemRefToEmitC/memref-to-emitc.mlir

```diff
@@ -45,3 +45,15 @@ func.func @alloca() {
   %0 = memref.alloca() : memref<4x8xf32>
   return
 }
+// -----
+
+// CHECK-LABEL: globals
+module @globals {
+  memref.global "private" constant @myglobal : memref<3x7xf32> = dense<4.0>
```

Comment:
Do we just drop the `private` here? Do we need to ensure this results in `internal` linkage in C++?